ui/page/components/wallet_account_selector.go

```diff
@@ -522,34 +522,32 @@ func (sm *selectorModal) accountValidator(accountIsValid func(*sharedW.Account)
 	return sm
 }
 
-func (sm *selectorModal) Handle() {
-	if sm.eventQueue != nil {
-		for _, selectorItem := range sm.selectorItems {
-			for selectorItem.clickable.Clicked() {
-				switch item := selectorItem.item.(type) {
-				case *sharedW.Account:
-					if sm.onAccountClicked != nil {
-						sm.onAccountClicked(item)
-					}
-				case sharedW.Asset:
-					if sm.onWalletClicked != nil {
-						sm.onWalletClicked(item)
-					}
+func (sm *selectorModal) Handle(gtx C) {
```

Comment:
check if source has been assigned.

Reply:
we don't need check nil for eventSource because it can't nil